package.json

```diff
@@ -89,6 +89,7 @@
     "moment-timezone": "^0.5.3",
     "ng-wig": "~2.3.4",
     "rx": "~4.0.8",
```

Comment:
You may have already done this, but ultimately we can remove the reference to `rx`